src/__tests__/Navbar.test.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+// eslint-disable-next-line import/extensions
+import Navbar from '../components/Navbar/Navbar';
+
+describe('Navbar', () => {
+  it('renders correctly', () => {
+    const { asFragment } = render(<Navbar />, { wrapper: BrowserRouter });
+
+    expect(asFragment(<Navbar />)).toMatchSnapshot();
+  });
+
+  it('contains the calculator link', () => {
+    render(<Navbar />, { wrapper: BrowserRouter });
+
+    expect(screen.getByText('Calculator')).toHaveClass('link');
+  });
+});
```

Comment:
- [ ] Hello there, team. So far, so good, however, I observed that one of your tests is failing; as necessary for this project, all of your tests should pass so that we can check whether you have settled everything correctly :wink: Please ensure that all of your tests are passed before returning when you are ready.  :100: 
![Screenshot from 2022-12-08 15-19-55](https://user-images.githubusercontent.com/88894525/206461289-903911b9-bab5-4142-876b-cec554c1a6e9.png)